### ISSUE TITLE
PLDM: pldmtool: Implement GetBIOSAttributeCurrentValueByHandle and SetBIOSAttributeCurrentValue

### ISSUE THREAD

**Issue by dkodihal** [ibm-openbmc/dev#1665]
To make these user-friendly, some of the protocol specific (attribute handles, looking up string and attribute value tables) must be handled by the tool.

GetBIOSAttributeCurrentValueByHandle

```
pldmtool bios GetBIOSAttributeCurrentValueByHandle --attribute <attribute_name>
```
Returned value should be a string (BIOSString or BIOSEnum),  or an integer (BIOSInteger), or failure.

SetBIOSAttributeCurrentValue

```
pldmtool bios SetBIOSAttributeCurrentValue --attribute <attribute_name> --value <string or integer>
```
Returns success or failure.

**Comment by mzipse** [comment]
Assigning to Deepak for now.  He can reassign once someone is ready to work on this.

**Comment by mzipse** [comment]
@wangzqbj   Hi John.  Wondering how this is coming?  Any idea when you might have this ready to push up code for review?

**Comment by wangzqbj** [comment]
@mzipse  

GetBIOSAttributeCurrentValueByHandle:
https://gerrit.openbmc-project.xyz/c/openbmc/pldm/+/30961

@lllllccccc  is working on this.(and the set part).   talked to him, maybe it will be ready for review next Tuesday

**Comment by mzipse** [comment]
Sounds good!  Thank you!

**Comment by mzipse** [comment]
@lllllccccc Just curious how you are doing with the remainder of this story?

**Comment by dkodihal** [comment]
Implemented at ce041e25567c58c5697117a6bd1c617aee9e1970
